### DISCUSSION TITLE
【React专项】

### DISCUSSION THREAD

**Discussion by swlws** [swustzzh/coder_books#35] [General]
React相关的零碎知识点

**Comment by swlws** [comment]
[智能组件和木偶组件](https://www.cnblogs.com/rik28/p/8676151.html)

智能组件：仅包含数据和行为，不涉及UI
木偶组件：仅做UI渲染

**Comment by swustzzh** [comment]
不是有React相关的了吗。。

**Reply by swustzzh** [reply]
https://github.com/swustzzh/coder_books/discussions/22
